docs/index.md

```diff
@@ -97,27 +111,59 @@ For more, see [user roles and permissions](https://docs.datastax.com/en/astra/do
 
 Astra DB Plugin for HashiCorp Vault is free. See the HashiCorp Platform Vault site for its [enterprise pricing](https://cloud.hashicorp.com/products/vault/pricing) details. 
 
+<a name="build-steps"></a>
 ## Build steps - optional
 
 If you elect to build the plugin from Go modules in our GitHub repo, follow these steps. Otherwise, you can use the provided binary.
```

Comment:
```suggestion
To build the plugin from Go modules in our GitHub repo, follow these steps. Otherwise, you can use the provided binary.
```

Reply:
We want to emphasize that building from source is optional.  I'll edit this part separately in a later commit.

Reply:
Edited in subsequent commit.